test/integration/start_stop_delete_test.go

```diff
@@ -339,7 +339,13 @@ func testPodScheduling(ctx context.Context, t *testing.T, profile string) {
 	}
 
 	// Arbitrary value set by some container runtimes. If higher, apps like MySQL may make bad decisions.
+	// Default for docker runtime: 1048576
+	// Containerd with docker driver: 1024
+	// Containerd with KVM driver: 1048576
 	expected := int64(1048576)
+	if ContainerRuntime() == "containerd" && DockerDriver() {
```

Comment:
would this fail in Jenkins ? if this is only prow-sepecifc we might wanna add an evironment detection to detect package and if detect.IsProw() is true expect different ulimit

Reply:
yes that could be the issue